content/en/hosting/4.x/data-migration-3x-docker-to-4x-k3s-multi.md

```diff
@@ -0,0 +1,166 @@
+---
+title: "Migration from Docker Compose CHT 3.x to 3-Node Clustered CHT 4.x on K3s"
+linkTitle: "Migration: 3.x Docker Compose to 4.x K3s (Multi-node)"
+weight: 1
+description: >
+  Guide to migrate existing data from CHT 3.x Docker Compose deployment to CHT 4.x clustered K3s deployment with 3 CouchDB nodes
+relatedContent: >
+---
+{{ < read-content file="hosting/4.x/_partial_migration_3x_docker_to_4x_k3s.md"  }}
+
+# Create directories on secondary nodes
+
+```shell
+ssh <user>@<node2-hostname> "sudo mkdir -p /srv/couchdb2/data/shards /srv/couchdb2/data/.shards"
+ssh <user>@<node3-hostname> "sudo mkdir -p /srv/couchdb3/data/shards /srv/couchdb3/data/.shards"
+```
+
+## Create values.yaml for K3s Deployment
+{{ < read-content file="hosting/4.x/_partial_values_explanation.md"  }}
+
+```yaml
+project_name: "your-namespace"
```

Comment:
It could be tough to keep the documentation _and_ the helm repo in sync. Can we, instead of inlining the whole contents of the values file somehow link to the current version of it?

Reply:
I have somehow struggled with this too. The original [values.yaml](https://github.com/medic/helm-charts/blob/main/charts/cht-chart-4x/values.yaml) in the helm charts is much larger and robust as it accounts for all the different scenarios. I originally thought that the comments on the yaml would be enough to guide people to the correct final configuration of what should be removed or what should be included. However, that hasn't been the case. That's why I decided to add example config of the various scenarios in the docs. Although not ideal, we'd have to remember to keep the docs in sync when we add/remove features from the helm-charts.